PocketKit/Sources/PocketKit/LoggedOut/LoggedOutViewModel.swift

```diff
@@ -13,7 +13,7 @@ enum LoggedOutAction {
 }
 
 class LoggedOutViewModel: ObservableObject {
-    weak var contextProvider: ASWebAuthenticationPresentationContextProviding?
```

Comment:
out of curiosity, what was the need for removing the `weak` modifier? It seems that `LoggedOutCoordinator` holds a strong reference to this view model and it also implements `ASWebAuthenticationPresentationContextProviding`, so we might want to keep the weak to avoid a cycle?

Reply:
When this is set to weak, its not retained and log in won't work